LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -1789,7 +1788,8 @@ LogFileMonitor::ReadLogFile(
                         //
                         // newLineBuffer was empty, so only print the found line.
                         //
-                        logWriter.WriteConsoleLog(decodedString.substr(0, found));
+                        //std::wstring decodedAll = Utility::ReplaceAll(decodedString.substr(0, found), L"\n", L"\n" + fileName);

```

Comment:
nit: if no longer in use, can just delete the line.

Reply:
This has been corrected. Thank you for pointing it out.

Reply:
oh, got it.